src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -612,14 +555,74 @@ public void showBasePanel(BasePanel bp) {
         tabbedPane.getSelectionModel().select(getTab(bp));
     }
 
+    public void init() {
+        sidePaneManager = new SidePaneManager(Globals.prefs, this);
+        sidePane = sidePaneManager.getPane();
+
+        tabbedPane = new TabPane();
+        tabbedPane.setTabDragPolicy(TabPane.TabDragPolicy.REORDER);
+
+        initLayout();
+
+        initKeyBindings();
+
+        initDragAndDrop();
+
+        //setBounds(GraphicsEnvironment.getLocalGraphicsEnvironment().getMaximumWindowBounds());
```

Comment:
Can this be removed or why is this (still) uncommented? What did it do?

Reply:
To be honest, I have no idea ;-) Probably to restore the window size on start. Feel free to investigate ;-)